### DISCUSSION TITLE
Here you can ask about the code in this repo.

### DISCUSSION THREAD

**Discussion by 0fie** [0fie/dotfiles#1] [Announcements]
## 👋 Welcome!
  Here you can ask about the code in this repo.

**Comment by Letgamer** [comment]
Why is ntfs3g installed as a system package under system/environment.nix ?
Do you have öa dualboot Setup with Windows?

**Reply by 0fie** [reply]
Hello! The reason is that I have an external HDD which uses NTFS.

I used to have a dual boot setup but then i got rid of Win 11. I should probably remove osProber too.

**Reply by Letgamer** [reply]
Okay, nice, what about making grub silent and using a custom grub theme?

**Reply by 0fie** [reply]
I currently don't know how to do that. Such improvements are at the bottom of my to-do list. I might look into it once I finish setting up Hyprland window rules.

**Reply by Letgamer** [reply]
Would you mind sharing what's on your TODO-List?^^

**Reply by 0fie** [reply]
Okay

- Configuring Plymouth with the catppuccin theme
- Replacing Waybar & Rofi with [AGS](https://github.com/Aylur/ags)
- And more. Some of the `.nix` files have TODO comments
